server/src/internalClusterTest/java/org/elasticsearch/snapshots/SnapshotStressTestsIT.java

```diff
@@ -842,8 +856,8 @@ private void startSnapshotter() {
                 boolean startedSnapshot = false;
                 try (TransferableReleasables localReleasables = new TransferableReleasables()) {
 
-                    // separate TransferableReleasables for blocking node restarts & index deletion so we can release these blocks and
-                    // permit data node restarts and index deletions as soon as the snapshot starts
+                    // separate TransferableReleasables for blocking node restarts so we can release these blocks and
+                    // permit data node restarts as soon as the snapshot starts
```

Comment:
IIUC, we do block index deletion for the whole process of taking full snapshots. It is not blocked when taking partial ones. Hence the change.